src/Php83/Php83.php

```diff
@@ -82,4 +83,116 @@ public static function mb_str_pad(string $string, int $length, string $pad_strin
                 return mb_substr(str_repeat($pad_string, $leftPaddingLength), 0, $leftPaddingLength, $encoding).$string.mb_substr(str_repeat($pad_string, $rightPaddingLength), 0, $rightPaddingLength, $encoding);
         }
     }
+
+    public static function str_increment(string $string): string
+    {
+        if ('' === $string) {
+            throw new \ValueError('str_increment(): Argument #1 ($string) cannot be empty');
+        }
+
+        if (!\ctype_alnum($string)) {
+            throw new \ValueError('str_increment(): Argument #1 ($string) must be composed only of alphanumeric ASCII characters');
+        }
+
+        if (\is_numeric($string)) {
+            $offset = stripos($string, 'e');
+            if ($offset !== false) {
+                $char = $string[$offset];
+                $char++;
+                $string[$offset] = $char;
+                $string++;
+
+                switch ($string[$offset]) {
+                    case 'f':
+                        $string[$offset] = 'e';
+                        break;
+                    case 'F':
+                        $string[$offset] = 'E';
+                        break;
+                    case 'g':
+                        $string[$offset] = 'f';
+                        break;
+                    case 'G':
+                        $string[$offset] = 'F';
+                        break;
+                }
+
+                return $string;
+            }
+        }
+
+        return ++$string;
+    }
+
+    public static function str_decrement(string $string): string
+    {
+        if ('' === $string) {
+            throw new \ValueError('str_decrement(): Argument #1 ($string) cannot be empty');
+        }
+
+        if (!\ctype_alnum($string)) {
```

Comment:
it'd be great to get rid of any calls to ctype so that the code doesn't rely 
preg_match on `^[a-zA-Z0-9]+$` should do it (same below)